src/Mvc/Mvc.RazorPages/src/Infrastructure/DefaultPageLoader.cs

```diff
@@ -88,6 +88,8 @@ private async Task<CompiledPageActionDescriptor> LoadAsyncCore(PageActionDescrip
                         }
                     },
             },
+            groupFinallyConventions: new Stack<Action<EndpointBuilder>>(),
+            finallyConventions: new Stack<Action<EndpointBuilder>>(),
```

Comment:
The order of `groupFinally` vs `finally` keeps switching for different APIs, really triggering my OCD